### DISCUSSION TITLE
Windows Hello IR Webcams instead of Intel Realsense?

### DISCUSSION THREAD

**Discussion by FennexFox** [opentrack/opentrack#1639] [Q&A]
Is this possible?

**Comment by sthalik** [comment]
There's no generic code in opentrack right now doing head tracking for a generic IR/stereographic webcam. Realsense does it, but only specifically for Intel hardware.

**Comment by Slion** [comment]
Actually EasyTracker was designed to be as generic as possible and works with Kinect v2 IR buffer. It does not do face tracking though just point tracking. In theory it could be built upon to support more IR devices and possibly even face tracking.
